test/test_backend.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
please remove the two files (above) that are being created through running the test from the PR